pkg/exchange/bitget/exchange.go

```diff
@@ -368,3 +370,59 @@ func (e *Exchange) CancelOrders(ctx context.Context, orders ...types.Order) erro
 	// TODO implement me
 	panic("implement me")
 }
+
+// QueryTrades queries fill trades. The trade of the response is in descending order. The time-based query are typically
+// using (`CTime`) as the search criteria.
+// If you need to retrieve all data, please utilize the function pkg/exchange/batch.TradeBatchQuery.
+//
+// ** StartTime is inclusive, EndTime is exclusive. If you use the EndTime, the StartTime is required. **
+// ** StartTime and EndTime cannot exceed 90 days. **
+func (e *Exchange) QueryTrades(ctx context.Context, symbol string, options *types.TradeQueryOptions) (trades []types.Trade, err error) {
+	if options.LastTradeID != 0 {
+		log.Warn("!!!BITGET EXCHANGE API NOTICE!!! The trade of response is in descending order, so the last trade id not supported.")
+	}
+
+	req := e.v2Client.NewGetTradeFillsRequest()
+	req.Symbol(symbol)
+
+	if options.StartTime != nil {
```

Comment:
if both StartTime and EndTime are given, you will need to check the time range?

Reply:
Please ensure the `end >= start`

Reply:
I was just thinking that the caller should use batch.TradeBatchQuery, so I missed this part.